.eslintrc.json

```diff
@@ -0,0 +1,36 @@
+{
+	"extends": "eslint:recommended",
+	"env": {
+		"browser": true,
+		"commonjs": true,
+		"amd": true
+	},
+	"globals": {
+		"global": "writable"
+	},
+	"rules": {
+		"semi": ["error", "always"],
+		"no-unused-vars": "off"
+	},
+	"overrides": [
+		{
+			"files": ["utils/**", "tests/**"],
```

Comment:
Just to confirm it lines everything that's _not_ in the overrides?

Reply:
The overrides mean "apply these linting rules to only these specific files".

I'm using it here so it knows that "utils" and "tests" are running in Node, and "tests" has some QUnit-related global variables (ESLint will otherwise complain about accessing undeclared global variables).